west-texas-oil/src/App.js

```diff
@@ -15,7 +15,7 @@ function App() {
           target="_blank"
           rel="noopener noreferrer"
         >
-          Learn React
+          West Texas Oil
```

Comment:
Looks like you only changed this one line @KenMan79 ?
I'll have to close this PR, when you have a more substantial change set submit a PR.
Or you can submit an Bug report to explain why only this one line change was necessary, and then re-submit the PR.